conf/zeppelin-site.xml.template

```diff
@@ -394,6 +394,12 @@
 </property>
 
 <property>
+  <name>zeppelin.notebook.default.owner.username</name>
+  <value></value>
+  <description>Set owner role by default in private mode</description>
```

Comment:
btw, did we say this isn't just for private mode any more?

Reply:
This is for both public and private, then I've fixed it.